src/SFA.DAS.EAS.Portal/Application/Services/IAccountDocumentService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using SFA.DAS.EAS.Portal.Database.Models;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EAS.Portal.Application.Services
+{
+    public interface IAccountDocumentService
+    {
+        Task<AccountDocument> Get(long id, CancellationToken cancellationToken = new CancellationToken());
```

Comment:
There's no point creating a real `CancellationToken`, that the caller has no access to. Default should be `default`, which is equivalent to CancellationToken.None.
```suggestion
        Task<AccountDocument> Get(long id, CancellationToken cancellationToken = default);
```

Reply:
Updated.